MvvmCross/Platforms/Uap/Presenters/MvxWindowsViewPresenter.cs

```diff
@@ -55,6 +58,14 @@ public override void RegisterAttributeTypes()
                    ShowAction = (view, attribute, request) => ShowRegionView(view, (MvxRegionPresentationAttribute)attribute, request),
                    CloseAction = (viewModel, attribute) => CloseRegionView(viewModel, (MvxRegionPresentationAttribute)attribute)
                });
+
+            AttributeTypesToActionsDictionary.Add(
+               typeof(MvxModalViewPresentationAttribute),
+               new MvxPresentationAttributeAction
+               {
+                   ShowAction = (view, attribute, request) => ShowModal(view, (MvxModalViewPresentationAttribute)attribute, (MvxViewModelInstanceRequest)request),
```

Comment:
Should not cast to MvxViewModelInstanceRequest here, but in the method

Reply:
I do not understand why attribute yes and request no. What criterion do you have?

Reply:
+1